qiskit_experiments/curve_analysis/curve_fit.py

```diff
@@ -15,12 +15,14 @@
 # pylint: disable = invalid-name
 
 from typing import List, Dict, Tuple, Callable, Optional, Union
+from uncertainties import correlated_values
```

Comment:
I think this code would be clearer if you just do `import uncertainties` so that when reading source code you know what library functions are coming from (np, scipy.optimize, uncertainties etc) eg `uncertaintes.correlated_values` and `uncertainties.ufloat`